src/Components/ShareImageModal/ShareImageModal.tsx

```diff
@@ -32,7 +32,7 @@ const ShareToRegionsModal = () => {
 
   if (!composeId) {
     handleClose();
-    return undefined;
```

Comment:
See comment above, `npm ci` should allow you to return undefined.

Reply:
It's working for eslint, not in my nvim, it's weird.

Reply:
Right, I updated my coc modules, it's now fine.